examples/redis/redis.yaml

```diff
@@ -62,6 +62,8 @@ data:
       type: Kubernetes
     gateway:
       controllerName: gateway.envoyproxy.io/gatewayclass-controller
+    extensionApis:
```

Comment:
this seems not related to an `redis`example, can you address some comment?

Reply:
this is the yaml used in e2e to update the startup config, I can start renaming and breaking things in separate files, would you prefer that ?

Reply:
yes, we can do that later